src/vs/code/electron-main/main.ts

```diff
@@ -306,7 +305,12 @@ function setupIPC(accessor: ServicesAccessor): TPromise<Server> {
 						promise = service.getMainProcessId()
 							.then(processId => {
 								logService.log('Sending some foreground love to the running instance:', processId);
-								allowSetForegroundWindow(processId);
+								try {
+									const { allowSetForegroundWindow } = <any>require.__$__nodeRequire('windows-foreground-love');
```

Comment:
@the-ress this is rather ugly, can we do what windows-mutex does? This native module is also only build for Windows and it still has a nice import 👍